### PR TITLE
Alien Slashes Now Attack Bodybag Occupants

### PR DESCRIPTION
**Pull request by Surrealaser**
## About The Pull Request

Alien Slashes Now Attack Bodybag Occupants

## Why It's Good For The Game

Alien Slashes Now Attack Bodybag Occupants

## Changelog
:cl:
fix: Alien slashes now hit bodybag occupants.
/:cl:

### CODE REVIEW

**Comment by lKiseki** [comment]
Yes. All my yes.

**Comment by RenGusta** [comment]
Won't this end up hitting hidden tarp users due to the tarp function being similar to bodybags?

**Comment by BraveMole** [comment]
but why? just open then slash?

**Comment by BraveMole** [comment]
Say it's because of tarp, because don't think you need to slash bodies in cry bags

**Comment by Surrealaser** [comment]
> Say it's because of tarp, because don't think you need to slash bodies in cry bags

Well yes, but no bag should provide any slash protection.

**Comment by RenGusta** [comment]
I still don't see whether or not this will impact bump slash, I can only hope it doesn't?

Otherwise this should probably be labelled balance, not fix.

**Comment by Surrealaser** [comment]
> I still don't see whether or not this will impact bump slash, I can only hope it doesn't?
> 
> Otherwise this should probably be labelled balance, not fix.

Bump slashing has literally nothing to do with this.

**Comment by EgorDinamit** [comment]
What about stasis bags? Can xenos open those?
If not, you could just hide in one and be revivable for eternity.

**Comment by Surrealaser** [comment]
> What about stasis bags? Can xenos open those?
> If not, you could just hide in one and be revivable for eternity.

Yes

File: code/game/objects/items/bodybag.dm [hunk 1]
```diff
@@ -202,6 +202,8 @@
 	if(opened)
 		return FALSE // stop xeno closing things
 	X.do_attack_animation(src, ATTACK_EFFECT_CLAW)
+	if(bodybag_occupant) //if there's an occupant, slash his ass
+		bodybag_occupant.attack_alien(X)
```
Comment (SplinterGP) [inline code/game/objects/items/bodybag.dm:206]:
```suggestion
	bodybag_occupant?.attack_alien(X)
```

**Review by SplinterGP — CHANGES_REQUESTED** [review]

**Pull request merged**